examples/file-server/test/file-server.unit.ts

```diff
@@ -0,0 +1,25 @@
+import { runEngineEnvironment } from '@wixc3/engine-test-kit';
+import Feature, { server } from '../feature/file-server.feature';
+import { expect } from 'chai';
+import fs from '@file-services/node';
+
+describe('Processing env test', () => {
+    it('reads directory', async () => {
+        const { dispose, engine } = await runEngineEnvironment({
```

Comment:
wrap it with a function that accepts the actual feature, and returns the typed api as well